src/components/ui/Panel.jsx

```diff
@@ -203,7 +203,6 @@ class Panel extends React.Component {
             className={classnames('panel', size, className, {
               'panel--white': white,
               'panel--holding': holding,
-              'panel--pj': dataSource === 'pages_jaunes',
```

Comment:
You also have to remove `dataSource` in the propTypes declaration and 10 lines above when you get it from props.